src/Nethermind/Nethermind.Merge.Plugin/Synchronization/PivotUpdator.cs

```diff
@@ -141,7 +141,7 @@ private async Task<bool> TrySetFreshPivot(CancellationToken cancellationToken)
 
         if (finalizedBlockHash is null || finalizedBlockHash == Keccak.Zero)
         {
-            if (_logger.IsInfo && _attemptsLeft % 10 == 0) _logger.Info($"Waiting for Forkchoice message from Consensus Layer to set fresh pivot block. {_attemptsLeft} attempts left");
```

Comment:
attempts left was pretty nice but if we keep int.max approach it does not have sense indeed. If You would plan to add some value like -1 to make it infinite instead of this approach, please rollback this change.

Reply:
If it will be infinite, then it will have even less sense, because infinite number of attempt will be always left. We can add one more variable and just increment it - it will be something like:
`Waiting for Forkchoice message from Consensus Layer to set fresh pivot block [{counter}s]`

And for users
`Waiting for Forkchoice message from Consensus Layer to set fresh pivot block [1s]`
`Waiting for Forkchoice message from Consensus Layer to set fresh pivot block [2s]`
...
`Waiting for Forkchoice message from Consensus Layer to set fresh pivot block [123s]`

WDYT?